src/backend/joanie/core/models/wishlist.py

```diff
@@ -0,0 +1,37 @@
+"""
```

Comment:
It's a possibility as for now we only have one table, but I prefer to separate `CourseWish` in a specific file `wishlist` because for me `accounts` it's more about authentification purpose. Also in the same idea, `Address` could be moved in another file.